go.mod

```diff
@@ -73,9 +72,9 @@ require (
 	github.com/xeipuuv/gojsonschema v0.0.0-20180618132009-1d523034197f
 	github.com/zchee/go-vmnet v0.0.0-20161021174912-97ebf9174097
 	go.opencensus.io v0.23.0
-	go.opentelemetry.io/otel v0.17.0
-	go.opentelemetry.io/otel/sdk v0.16.0
-	go.opentelemetry.io/otel/trace v0.17.0
+	go.opentelemetry.io/otel v1.0.0-RC1
```

Comment:
Just curious, is there a reason we can't go to `v1.2.0`?

Reply:
that was done automatically

Reply:
but thats a good idea I changed it to 1.2.0